src/Components/Navbar.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
+import '../Style/navbar.css';
 
 const NavBar = () => (
-  <nav>
-    <h1>Bookstore</h1>
+  <nav className="head-nav">
     <ul className="navbar">
-      <li>
+      <li className="head-link">Bookstore CMS</li>
+      <li className="link">
         <a href="/">Books</a>
       </li>
-      <li>
+      <li className="link">
         <a href="/categories">Categories</a>
       </li>
     </ul>
+    <i className="fas fa-user user" />
```

Comment:
- [ ] Nice design. Kindly make a circle border to this icon so as to match the guiding template.

### Your Design
![your-img-icon](https://github.com/KAYEMBAJONATHAN/Book_s/assets/26597432/454e3f1c-3f22-4d38-aa6f-b7a40a0cc3e7)

### Guiding Template
![temp-img-icon](https://github.com/KAYEMBAJONATHAN/Book_s/assets/26597432/ef6e3853-672e-4ca5-8455-8e41ca97aac9)